go-controller/pkg/node/management-port.go

```diff
@@ -24,6 +24,13 @@ type ManagementPort interface {
 	// CheckManagementPortHealth checks periodically for management port health until stopChan is posted
 	// or closed and reports any warnings/errors to log
 	CheckManagementPortHealth(cfg *managementPortConfig, stopChan chan struct{})
+	// Not all management ports have an IP address. Some cases where a management port would not have an IP address
```

Comment:
The comment could be made exhaustive: "Currently, the management port doesn't have an IP in the following cases"

Reply:
Good point! Thanks.